ConnectionsPanel.js

```diff
@@ -420,4 +382,110 @@ ToolsButton.propTypes = {
   count:         PropTypes.number,
 };
 
+const ConnectionsPanelSection = ({ title, children }) => {
+  const { themeStr, interfaceLanguage } = useContext(GlobalStateContext);
+  const theme = getTheme(themeStr);
```

Comment:
`theme` exists in GlobalStateContext. `themeStr` is unnecessary in this case. See other usages of `useContext(GlobalStateContext);`

Reply:
const { theme, interfaceLanguage } = useContext(GlobalStateContext);
theme.tertiaryText
throws error
this is way to get theme is also the usage wat in ToolsButton

Reply:
You're right. I forgot there's a hook in Hooks.js called `useGlobalState`. This can be used like this: 

```javascript
const { theme, interfaceLanguage } = useGlobalState();
```